uint8 ZERO_POWER_BEHAVIOR_COAST = 1
uint8 ZERO_POWER_BEHAVIOR_BRAKE = 0
uint8 behavior
---
